### ISSUE TITLE
iam/api-client - missing method in latest build.

### ISSUE THREAD

**Issue by lesv** [GoogleCloudPlatform/java-docs-samples#4357]
Found in #4351 @donmccasland PTAL @chingor13 FYI

# Java 11
```
------------------------------------------------------------
- testing iam/api-client
------------------------------------------------------------
[ERROR] Tests run: 6, Failures: 0, Errors: 3, Skipped: 0, Time elapsed: 0.806 s <<< FAILURE! - in iam.snippets.AccessTests
[ERROR] iam.snippets.AccessTests.testTestPermissions  Time elapsed: 0.621 s  <<< ERROR!
java.lang.NoSuchMethodError: 'boolean com.google.api.client.http.HttpTransport.isMtls()'
	at com.google.api.services.cloudresourcemanager.CloudResourceManager$Builder.chooseEndpoint(CloudResourceManager.java:5501)
	at com.google.api.services.cloudresourcemanager.CloudResourceManager$Builder.<init>(CloudResourceManager.java:5534)
	at iam.snippets.TestPermissions.createCloudResourceManagerService(TestPermissions.java:75)
	at iam.snippets.TestPermissions.testPermissions(TestPermissions.java:41)
	at iam.snippets.AccessTests.testTestPermissions(AccessTests.java:119)
```

**Comment by lesv** [comment]
Note either the client library is bad, or the sample needs to be rewitten

**Comment by lesv** [comment]
Seems to need google-http-client 1.38.0, should be fixed in #4365

**Comment by lesv** [comment]
Fixed
